### DISCUSSION TITLE
Google calendar with many sub calendars

### DISCUSSION THREAD

**Discussion by RockFan89** [kewisch/gdata-provider#547] [Support]
Hi,
I've created in my Google calendar 10 additional calendars for family members and different purposes. Recently I found this nice add-on for Thunderbird and immediately added it.
Unfortunately, the result only shows 7 out of the 11 calendars. Is this a limitation of the add-on?

Thanks and regards
Dieter

**Comment by kewisch** [comment]
I'm not exactly sure, I don't recall anything specific. In the wiki there are some steps to enable debugging, can you try that and see if all are returned in the API response?

**Reply by RockFan89** [reply]
The Error Console shows exactly the calndars that are visible to me. The 4 sub-calendars don't show up in any entry of the debug. There's also no number that shows a limitation. The only error about calender in the log  appear before connecting to google:

11:33:06.053 Assert failed: Calendar manager initialised calendars before loadCalendarComponent ran on the first 3pane window. This should not happen.
 calUtils.jsm:109
    ASSERT resource:///modules/calendar/calUtils.jsm:109
    loadCalendarComponent chrome://calendar/content/calendar-chrome-startup.js:40
    _loadComponentsAtStartup chrome://messenger/content/msgMail3PaneWindow.js:713
    _delayedStartup chrome://messenger/content/msgMail3PaneWindow.js:697
    _delayedStartup self-hosted:1115

**Reply by kewisch** [reply]
Those errors seem to be from Thunderbird proper. If the missing calendars are not in the API response, not sure I can show them. Are they hidden in the Google Calendar web interface somehow? If you are proficient with source code, you could potentially try adding the `showHidden` parameter.
